src/main/java/com/squareup/javapoet/Util.java

```diff
@@ -109,4 +110,19 @@ public static void requireExactlyOneOf(Set<Modifier> modifiers, Modifier... mutu
   public static boolean hasDefaultModifier(Collection<Modifier> modifiers) {
     return DEFAULT != null && modifiers.contains(DEFAULT);
   }
+
+  public static String identifier(String part) {
```

Comment:
The only usage is quite simple. :smile: Only put the method into `Util` because of it's easier to test. Move back to `JavaFile` and make it private? It's not intended to be a general purpose method.